utilities.c

```diff
@@ -425,12 +425,14 @@ uint64_t iio_read_counter_us(void)
 	QueryPerformanceCounter(&cnt);
 
 	value = (1000000 * cnt.QuadPart) / freq.QuadPart;
-#else
+#elif CLOCK_GETTIME_AVAILABLE
 	struct timespec ts;
```

Comment:
do you need to protect the 

```
#include <time.h>
```

at the top of this file?

Reply:
It would be better to protect it.

Reply:
That's now fixed.